### ISSUE TITLE
ibmcloud sl block volume-list needs --columns clarifying and/or examples 

### ISSUE THREAD

**Issue by pstatham** [IBM-Bluemix-Docs/cli#84]
https://cloud.ibm.com/docs/cli/reference/ibmcloud?topic=cloud-cli-sl-block-storage#sl_block_volume_list

Options currently reads:
```
--columns
    Columns to display, options are: id,username,datacenter,storage_type,capacity_gb,bytes_used,ip_addr,created_by,notes.
```

But using commas between the fields is apparently unsupported.
```
$ ibmcloud sl block volume-list --columns id,username,notes
Incorrect Usage: --columns 'id,username,notes' is not supported.
```

Through trial-and-error, I eventually figured out that in order to show multiple columns, the `--columns` option needs to be specified for each one. This is counter-intuitive since "columns" is plural, which usually means it will take multiple values.
```
$ ibmcloud sl block volume-list --columns id --columns username --columns notes
id         username              notes
...
```

Also, there are other `ibmcloud sl block` commands that take a `--columns` option that should get the same clarification and/or examples.

**Comment by boyang9527** [comment]
thanks for reporting this.. Will change it to --column  and add examples

**Comment by jpkessle** [comment]
@boyang9527 - Hi Bo, do we have an eta for this change? Is it scheduled for the next 0.16.0 release ? Do you have a development issue open to track this from your end ?

**Comment by archerzz** [comment]
@jpkessle This will come in the bug fix release `0.15.1`. It should come soon.

**Comment by jpkessle** [comment]
Changes are being worked on and scheduled for a future CLI release. Closing issue.
